pyomo/mpec/tests/t11_mpec.simple_disjunction.txt

```diff
@@ -50,7 +50,7 @@ cc : Size=1, Index=None, Active=True
 
     1 Disjunction Declarations
         complements : Size=1, Index=None, Active=True
-            Key  : Lower : Body                                                                     : Upper : Active
-            None :   1.0 : cc.expr1.indicator_var + cc.expr2.indicator_var + cc.expr3.indicator_var :   1.0 :   True
+            Key  : Disjuncts                            : Active : XOR
+            None : ['cc.expr1', 'cc.expr2', 'cc.expr3'] :   True : True
```

Comment:
This move to list from a sum is expected?

Reply:
It's a move from a `Constraint-like` to a list, with a flag for whether it's an XOR disjunction vs. an OR disjunction. It introduces support for OR disjunctions, whereas the old approach only supported XOR disjunctions.